### ISSUE TITLE
The relative module was not found

### ISSUE THREAD

**Issue by BeeeQueue** [skiptirengu/anitomy-js#15]
I'm trying to use this with webpack via `@vue/cli` and get the following error when building:

```
This relative module was not found:

* ./build/Release/anitomy-js in ./node_modules/anitomy-js/index.js
```

Other native modules work fine, like `node-sass` or `register-scheme` but this one fails to load.

It seems like changing `index.js:2` to 
```diff
-const anitomy = require('./build/Release/anitomy-js')
+const anitomy = require('./build/Release/anitomy-js.node')
```
fixes it

**Comment by BeeeQueue** [comment]
I had to add `.node` as a default extension in my webpack config.

**Comment by skiptirengu** [comment]
I'll change the require line to include the file extension. Thanks for the heads up!

**Comment by skiptirengu** [comment]
Fixed on v4.0.2.
